bot/extensions/8ball.py

```diff
@@ -0,0 +1,38 @@
+import discord
+from discord.colour import Color
+from discord.ext.commands import Cog, command
+from discord import Embed
+import random
+
+
+class eightball(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='eightball', help="Ask a question and be answered by Grace", aliases=['8ball'])
+    async def eightball(self, ctx):
+
+        responses = ['Hell no.', 'Prolly not.', 'Idk bro.',
```

Comment:
This is a lot of hard-coded data. For now it's accepted but soon I will have added a configuration system for extensions. The purpose of this config system will be to allow each extension, like this, to have their own configurations and list of data like this.

For now, it's not done so just keep in mind that if I'm done implementing the extension config system before this PR is approved, I will ask you bring the changes for that. If it's approved before, I will take the responsibility upon me to make the changes.